src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 1; i <= 3; i++) {
+            String ballNumber = String.valueOf(lottery.getRandomBall().getNumber());
+            String ballCollor = lottery.getRandomBall().getColor();
+            System.out.println("Ball " + i + " number :" + ballNumber + " color : " + ballCollor);
```

Comment:
1. you creating 2 random balls per 1 loop iterating, its not accepted
2. just create toString method in Ball class instead of this
```suggestion
            System.out.println(lottery.getRandomBall());
```